MAINTAINERS.md

```diff
@@ -0,0 +1,161 @@
+serve as a wiki for maintainers to link to / include resources.
+
+
+## Current Maintainers
+
+The current maintainers are:
+
+| name | focus areas |
+------------------
+|   @achow101  | wallet, PSBTs |
+|   @fanquake | general, tooling, build system |
+|   @hebasto | UI/UX |
+|   @laanwj | general |
+|   @marcofalke | general, tests, QA |
+|   @glozow | general, transaction relay and mempool policy |
+
+
+
+## Regular Contributors, Scoped Maintainers, General Maintainers, and Lead Maintainers
+
+The title of Maintainer in Bitcoin is more of a "janitorial responsibility"
+than a position of privilege. However, the role does include permissions on the
+repository, and connotes a general sense of stewardship over the project.
+
+
+Largely, the role of maintainers is to decide to merge Pull Requests.
+
+Whether a pull request is merged into Bitcoin Core rests with the project merge
+maintainers.
+
+Maintainers will take into consideration if a patch is in line with the general
+principles of the project; meets the minimum standards for inclusion; and will
+judge the general consensus of contributors. The review process that
+establishes general consensus is detailed in
+[CONTRIBUTING.mg](/CONTRIBUTING.md).
+
+Thus maintainers role is not directly elevated above that of regular
+contributors, as maintainers serve mostly as arbiters of if changes are
+acceptable to the contributors to the project in aggregate.
+
+The role of the Maintainer is currently evolving with respect to "Scoped
+Maintainers". The concept of a scoped maintainer is that it is helpful to add
+maintainers who are subject matter experts in a particular area of the code,
+but perhaps not in other areas. As such, maintainers with an explicit scope
+typically will avoid merging PRs in areas outside their scope, but may do so on
+occasion. As an individual maintainer's experience levels in the project
+increase, they may change their focus from scoped to general, but this is not
+currently an explicit designation (all maintainers should be considered to be
+general at this time). As a rough measure, scoped maintainers should avoid
+"stepping on the toes" of work that is being guided by another maintainer or
+contributor.
+
+Historically, one Bitcoin's maintainers was a designated "Lead Maintainer",
+which was most recently @laanwj. However, [in a blog
+post](https://laanwj.github.io/2021/01/21/decentralize.html), @laanwj stated
+that they wanted to take more of a background role as a maintainer and have not
+"passed the baton" to a successor as a Lead Maintainer, and instead focused on
+decentralizing the maintainership ecosystem. As such, while the unofficial
+title of Lead Maintainer still is held by @laanwj, it should be considered a
+bug-not-feature if it confers any additional responsibility or privilege over
+other General Maintainers, and may be explicitly deprecated in the future.
+
+## Self Merges
+
+Typically, a PR authored by a maintainer will be merged by a different
```

Comment:
This is new policy, which probably would have to be iterated on a few times to make sure it strikes the right balance of accountability and light-weight process.